Sources/Swift/Persistence/SentryScopeContextPersistentStore.swift

```diff
@@ -0,0 +1,83 @@
+@_implementationOnly import _SentryPrivate
+
+@objcMembers
+class SentryScopeContextPersistentStore: NSObject {
+    private let fileManager: SentryFileManager
+
+    init(fileManager: SentryFileManager) {
+        self.fileManager = fileManager
+    }
+
+    // MARK: - Context
+
+    func readPreviousContextFromDisk() -> [String: [String: Any]]? {
+        let fm = FileManager.default
+        guard fm.fileExists(atPath: contextFileURL.path) else {
```

Comment:
`h`: That should be the `previousContextFilePath` if I'm not mistaken. We should add a test for this, because a test should fail cause we're reading the current `contextFileURL` instead of the previous one.

Reply:
I agree, thanks for pointing this out.

Yes I'll add more tests